assets/styles/css/styles.css

```diff
@@ -19,74 +28,164 @@ a {
   color: black;
 }
 
-nav {
-  flex: 1 0 auto;
+button {
+  border: none;
 }
 
-footer {
-  flex: 0 0 auto;
+._ibg {
+  position: relative;
 }
 
-input[type="email"] {
+._ibg img {
+  position: absolute;
   width: 100%;
-  height: 25px;
-  border: none;
-  outline: none;
-  font-family: Raleway;
+  height: 100%;
+  top: 0;
+  left: 0;
+  object-fit: cover;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+.button {
+  flex: 0 0 auto;
+  width: 200px;
+  height: 65px;
+  text-transform: uppercase;
+  font-weight: 900;
   font-size: 21px;
-  line-height: 25px;
+  line-height: 119%;
```

Comment:
It's better to define exact value for that instead of percent.  `height: 65px;` height is not dynamic, that's why it doesn;t make sence to use percent for line-height